src/styles/utils/_reset.scss

```diff
@@ -0,0 +1,14 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+}
+
+body,
+h1,
+p {
+  margin: 0;
+}
```

Comment:
reset margin and paddings for `ul` as well, it could help with some failed tests